nabweb/locale/fr_FR/LC_MESSAGES/django.po

```diff
@@ -215,69 +214,98 @@ msgstr ""
 
 #: templates/nabweb/help.html:17
 msgid "Subscribe to the newsletter."
-msgstr "Abonnez vous à la lettre d'information."
+msgstr "Abonnez-vous à la lettre d'information."
 
 #: templates/nabweb/help.html:18
 msgid ""
+"<a target='_blank' href='https://dsc.gg/nabaztagtagtag' rel='noopener "
+"noreferrer'>Discord Nabaztag & Tag:Tag:Tag</a>"
+msgstr ""
+"<a target='_blank' href='https://dsc.gg/nabaztag' rel='noopener "
+"noreferrer'>Discord Nabaztag & Tag:Tag:Tag</a>"
+
+#: templates/nabweb/help.html:18
+msgid "Modern place to discuss."
+msgstr "Salon de discussoin."
```

Comment:
I do not think we should add a reference to Discord in the help page.
If needed (will it live long?..), it should rather be referenced from the tagtagtag.fr project page.